modules/nw-egress-ips-about.adoc

```diff
@@ -19,30 +18,136 @@ The {product-title} egress IP address functionality allows you to ensure that th
 
+An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations.
+
+ifdef::openshift-sdn[]
 An egress IP address is implemented as an additional IP address on the primary network interface of a node and must be in the same subnet as the primary IP address of the node. The additional IP address must not be assigned to any other node in the cluster.
+endif::openshift-sdn[]
+
+[IMPORTANT]
+====
+Egress IP addresses must not be configured in any Linux network configuration files, such as `ifcfg-eth0`.
+====
 
 [id="nw-egress-ips-platform-support_{context}"]
 == Platform support
 
 Support for the egress IP address functionality on various platforms is summarized in the following table:
 
-[IMPORTANT]
-====
-The egress IP address implementation is not compatible with Amazon Web Services (AWS), Azure Cloud, or any other public cloud platform incompatible with the automatic layer 2 network manipulation required by the egress IP feature.
-====
-
 [cols="1,1",options="header"]
 |===
 
 | Platform | Supported
 
 | Bare metal | Yes
-| vSphere | Yes
+| VMWare vSphere | Yes
 | {rh-openstack-first} | No
-| Public cloud | No
+| Amazon Web Services (AWS) | Yes
+| Google Cloud Platform (GCP) | Yes
+| Microsoft Azure | Yes
 
 |===
 
+[IMPORTANT]
+====
+The assignment of egress IP addresses to control plane nodes with the EgressIP feature is not supported on a cluster provisioned on Amazon Web Services (AWS). (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039656[*BZ#2039656*])
+====
+
+[id="nw-egress-ips-public-cloud-platform-considerations_{context}"]
+== Public cloud platform considerations
+
+For clusters provisioned on public cloud infrastructure, there is a constraint on the absolute number of assignable IP addresses per node. The maximum number of assignable IP addresses per node, or the _IP capacity_, can be described in the following formula:
+
+[source,text]
+----
+IP capacity = public cloud default capacity - sum(current IP assignments)
+----
+
+While the Egress IPs capability manages the IP address capacity per node, it is important to plan for this constraint in your deployments. For example, for a cluster installed on bare metal infrastructure with 8 nodes you can configure 150 egress IP addresses. However, if a public cloud provider limits IP address capacity to 10 IP addresses per node, the total number of assignable IP addresses is only 80. To achieve the same IP address capacity in this example cloud provider you would need to allocate 7 additional nodes.
+
+To confirm the IP capacity and subnets for any node in your public cloud environment, you can enter the `oc get node <node_name>` command. The `cloud.network.openshift.io/egress-ipconfig` annotation includes capacity and subnet information for the node.
+
+The annotation value is an array of objects with fields that provide the following information:
```

Comment:
We should maybe mention that for OpenShift 4.10 (possibly also 4.11 and later releases) this array will always be of length 1. This is because the egress IP assignment is always done against the primary network interface (or first one retrieved). We have Epics in the backlog intended to change this for future releases, but we have not begun the work yet and I have no idea when it will happen. We will however need to remember to change this in the future once we do implement that feature, so that it doesn't say that the array is of length 1 forever

Reply:
@alexanderConstantinescu, we generally try to avoid forecasting what the future might hold, so I'll just mention that it's an array with a single entry. We'll update the docs in a future release if this does change.